### ISSUE TITLE
A blank line should always separate block quotes

### ISSUE THREAD

**Issue by artragis** [zestedesavoir/zmarkdown#307]
The minimum markdown used : 

```
> Bonjour !
Source: Une personne.

> Au revoir !
Source: Une autre personne.
```

# result 

```
<figure><blockquote>
<p>Bonjour !
Source: Une personne.</p>
<p>Au revoir !</p>
</blockquote><figcaption>Une autre personne.</figcaption></figure>
```
# expected

```
<figure><blockquote>
<p>Bonjour !</p>
</blockquote><figcaption>Une personne.</figcaption></figure>
<figure><blockquote><p>Au revoir !</p>
</blockquote><figcaption>Une autre personne.</figcaption></figure>
```

**Comment by vhf** [comment]
This is weird, `gfm` should already take care of this.

**Comment by arobase-che** [comment]
Hi \o

I have some time to study that case.

**tl;dr**: The problem is [here](https://github.com/remarkjs/remark/blob/master/packages/remark-parse/lib/tokenizer.js#L307), line 307 of the file tokenizer.js from remark-parse.

It's a remark problem since : 

~~~js
const unified = require('unified');
const markdown = require('remark-parse');
const inspect = require('unist-util-inspect');

const processor = unified()
  .use(markdown, {gfm: true});

const ast = processor.parse(`

> Bonjour !
Source: Une personne.

> Au revoir !
Source: Une autre personne.
`);
console.log(inspect(ast));
~~~

Yield : 
~~~
root[1] (1:1-8:1, 0-79)
└─ blockquote[2] (3:1-7:28, 2-78)
   ├─ paragraph[1] (3:3-4:22, 4-35)
   │  └─ text: "Bonjour !\nSource: Une personne." (3:3-4:22, 4-35)
   └─ paragraph[1] (6:3-7:28, 39-78)
      └─ text: "Au revoir !\nSource: Une autre personne." (6:3-7:28, 39-78)
~~~

Should I open an issue and submit a PR in remark referencing [gfm#213](https://github.github.com/gfm/#example-213) ?

**Comment by artragis** [comment]
I think you should, yes.

**Comment by arobase-che** [comment]
Ok done. remark/remarkjs#415

It is gfm#220 not gfm#213 btw.

**Comment by arobase-che** [comment]
Fixed in remarkjs/remark ! ^^

We need to update remark to take into account the fix :)  
But we have to wait for a next version :(

**Comment by vhf** [comment]
Nice, thanks!
@arobase-che do you want to work on upgrading remark? Anyone else? I probably won't have time but I'd be happy to help.

We don't need to wait for a remark release containing this bugfix to work on this.

**Comment by arobase-che** [comment]
Ok.

I will try to update remark so. ^^

**Comment by TitiAlone** [comment]
Solve by today's release (zmarkdown 8.0.0)!
